test-specifications/token-service/customFee.md

```diff
@@ -0,0 +1,57 @@
+# Custom Fee
+
+Custom fees can be added to tokens that will be charged users automatically when being transferred based on a variety of parameters. These fees can be specified when a token is created (`TokenCreateTransaction`) or added/updated at a later time (`TokenFeeScheduleUpdateTransaction`).
+
+## Custom Fee Object Definition
+
+| Parameter Name      | Type         | Required/Optional | Description/Notes                                                                                                                                                                                                                                                                          |
+|---------------------|--------------|-------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| feeCollectorAccount | string       | required          | The ID of the account to which all fees will be sent when assessed.                                                                                                                                                                                                                        |
+| feeCollectorsExempt | bool         | required          | Should the token's treasury account and fee collector account be exempt from being charged fees when transferring the token?                                                                                                                                                               |
```

Comment:
This isn't quite accurate, from the specification:
> The treasury account for a token, and the account identified by the `fee_collector_account_id` field of this `CustomFee` are always exempt from this custom fee to avoid redundant and unnecessary transfers.

What this boolean does is exempt all _other_ fee collector accounts, so accounts identified as collector accounts in _other_ custom fee definitions for the same token are _also_ exempt if this flag is set.
From the specification:
> If this value is `true` then the account(s) identified in `fee_collector_account_id` for _all_ custom fee definitions for this token type SHALL also be exempt from this custom fee. This behavior is specified in HIP-573.

Reply:
Understood, I updated the language here to capture this more accurately

Reply:
Much better.
Side-note: Github won't let me resolve these comments for some reason...